app/models/post.rb

```diff
@@ -1,2 +1,6 @@
 class Post < ApplicationRecord
+  belongs_to :user
+  validates :user_id, presence: true
```

Comment:
By default, Rails 5 will make sure the foreign key is present when you declare a belongs_to association, so you do not have to validate this attribute's presence 😄